genesyscloud/resource_genesyscloud_outbound_messagingcampaign_test.go

```diff
@@ -30,18 +30,17 @@ func TestAccResourceOutboundMessagingCampaign(t *testing.T) {
 		contactListResource   = generateOutboundContactList(
 			contactListResourceId,
 			contactListName,
-			"",
-			"",
+			nullValue,
```

Comment:
for understanding purpose. why is this changed from empty string to nullValue ?

Reply:
Previously, the generate function was creating the resource as a string, and to exclude certain fields from the resource, the function contains an if statement to check if the string was empty. If it was empty, then the line would be omitted. It looked something like this:
```golang
func generateResource(name string) string {
    if name != "" {
        name = fmt.Sprintf(`name = "%s"`, name)
    }
    return fmt.Sprintf(`
    resource "example" "resource" {
        %s
    }
`, name)
}
```

Now, instead, I'm passing in `nullValue` (which is "null"). Setting the field to null does the exact same thing as omitting the field altogether, and this approach also makes the generate func more readable.

```golang
func generateResource(name string) string {  
    return fmt.Sprintf(`
    resource "example" "resource" {
        name = %s
    }
`, name)
}
```

Passing in `nullValue` evaluates to `name = null`, and to add a value you need to wrap the data in quotation marks when passing it to the function - this is what the `strconv.Quote()` func does